### ISSUE TITLE
AVT camera not found

### ISSUE THREAD

**Issue by llorban** [strawlab/flydra#35]
I am trying to run a camnode launch file. The launch looks like so: 
```
<launch>
  <node name="Cam1" pkg="camiface_ros" type="camnode">
    <param name="device_guid" type="str" value="AVT-02-2020C-06767" />
  </node>
  <node name="Cam2" pkg="camiface_ros" type="camnode">
    <param name="device_guid" type="str" value="AVT-02-2020C-07056" />
  </node>
</launch>
```

And this is what the log looks like: 
```
^[[0m[ INFO] [1531769131.179236474]: host timestamps OFF^[[0m
^[[33m[ WARN] [1531769132.255169176]: Selected camera AVT-02-2020C-07056 (-1) not found^[[0m
```

When running flydra in self-discovery mode, the cameras are found and detected as: 
```
beelab@beelab:~$ rosrun ros_flydra camnode
[INFO] [1531768934.973439]: ROS name: /flydra_camnode
[INFO] [1531768935.003848]: choosing camera guid: Allied Vision Technologies-02-2020C-07056 (cam_id: 0)
[INFO] [1531768935.004234]: choosing camera guid: Allied Vision Technologies-02-2020C-07058 (cam_id: 1)
[INFO] [1531768935.004501]: choosing camera guid: Prosilica-02-2020C-06767 (cam_id: 2)
```

This Flydra instance is a fresh installation on July 16th. 

Thank you!

**Comment by astraw** [comment]
Did you try using `Allied Vision Technologies-02-2020C-07056` instead of `AVT-02-2020C-07056` in your launch file? What happens if you do that?

**Comment by llorban** [comment]
That's what I had originally, but it didn't work either. When running the self-discovery process, I somewhere noticed that the cameras were renamed as "AVT" and underscores in the serial number. I tried that naming scheme but it also didn't work.

**Comment by astraw** [comment]
Going back to your original issue which is, as I understand it, "AVT camera not found". However, `rosrun ros_flydra camnode` shows that AVT cameras are chosen. For example: `choosing camera guid: Allied Vision Technologies-02-2020C-07056`. (I do not know where the Prosilica camera is coming from given that your launch file has only 2 non-Prosilica cameras in it.) What makes you think the AVT camera is not found?

If you save data, data from how many cameras is in the saved .h5 file? (e.g. using `flydra_analysis_plot_timeseries_2d_3d <FILENAME.H5>` https://flydra.readthedocs.io/en/latest/gallery.html#timeseries-of-2d-data )

**Comment by llorban** [comment]
Yes it's true that `rosrun` works fine, but I can't get the camnode launch file to work. I figure eventually it would be good to specify parameters which will need launch files instead of `rosrun`. 

Yds I do get data from all 3 cameras in the h5 file.

**Comment by astraw** [comment]
In your launch file, you are trying to launch the program `camnode` from the package `camiface_ros`, but you want ros_flydra `camnode` from the `ros_flydra` package.
